src/main/java/io/personium/client/BoxManager.java

```diff
@@ -110,4 +115,18 @@ public Box retrieve(String name) throws DaoException {
         return new Box(accessor, json, UrlUtils.append(accessor.getCurrentCell().getUrl(), name));
     }
 
+    /**
+     * This method performs recursively delete box
+     * @param boxName boxName to be delete
+     * @return PersoniumResponse 
+     * @throws DaoException Library Exception
+     */
+    public PersoniumResponse recursiveDelete(String boxName) throws DaoException {
```

Comment:
Please add the explanation of the added content to the Pull Request next time.

Reply:
I'm sorry for lack of explaination